sections/semantics-forms.include

```diff
@@ -8543,6 +8543,7 @@ You cannot submit this form when the field is incorrect.</samp></pre>
     <dd>If the element has a <{option/label}> attribute and a <{option/value}> attribute: <a>Nothing</a>.</dd>
     <dd>If the element has a <{option/label}> attribute but no <{option/value}> attribute: <a>Text</a>.</dd>
     <dd>If the element has no <{option/label}> attribute: <a>Text</a> that is not <a>inter-element whitespace</a>.</dd>
+    <dd><a>Nothing</a>; it is also permissable for the element to have no <{option/label}> and no <{option/value}> attributes.</dd>
```

Comment:
I think you can just remove the `that is not <a>inter-element whitespace</a>.`

Chedck out the text for inter-element whitespace but I think you can just remove that bit of text and there isn't a need for the Nothing Content type
http://glowsoft/html/single-page.html#inter-element-whitespace